_posts/2023-07-07-long-running-operation-and-component-template.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "New for OpenSearch Dashboards: Long-running operation notifications and component templates"
+authors:
+  - ihailong
+  - gbinlong
+  - suzhou
+  - gzhichao
+  - kolchfa
+date: 2023-07-07
+categories:
+  - technical-post
+meta_keywords: index management, index operation UI, Notifications, OpenSearch Dashboards
+meta_description: Transform long-running operations, including Resize/Open/Force merge, into tasks and set up related notifications when a specific task finishes or fails. Simplify cluster operations with Index Management UI enhancements that enable you to manage component templates in a more user-friendly way.
```

Comment:
Updated meta: 
**Meta_keywords:** OpenSearch index state management, index management UI, OpenSearch dashboards
**Meta_description:** Learn how OpenSearch simplifies cluster operations with index management UI enhancements that enable you to manage component templates in a user-friendly way.